MyApp/app/src/main/java/com/example/janari/myapp/MainActivity.java

```diff
@@ -25,15 +26,33 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
+        // Display the current date
         TextView dateView = (TextView)findViewById(R.id.date_today);
         setDate(dateView);
 
+        // Get calculated daily sum and displays it in Daily sum field
+        String value = getIntent().getStringExtra("key");
+        TextView textValue = (TextView) findViewById(R.id.daily_sum);
+        textValue.setText(value);
+
+        // This button exists because at the moment I don't have another option to communicate with other views
```

Comment:
Use `// TODO` comments instead. This will make finding incomplete or temporary thoughts/solutions much easier. Later on you can view all your TODO's and make sure that you'll either fix or remove them